src/model/Document.h

```diff
@@ -88,7 +88,7 @@ class Document
 
 	void lock();
 	void unlock();
-	bool tryLock();
+	bool try_lock();
```

Comment:
try to avoid making personal preference changes in code you otherwise aren't changing.